### PR TITLE
Add page about lightning quacks

### PR DESCRIPTION
**Pull request by katiewalker**
Adds a page with information about Lightning Quacks, including a link to register.

Just used a Google form for registration, don't have time to deal with Firebase.

Don't merge yet. We'll fill in the space on the LHS with a graphic, once Rachel's friend is done.

### CODE REVIEW

**Comment by ZacharyForman** [comment]
Lgtm merge at will

**Comment by katiewalker** [comment]
> Lgtm merge at will

Pls.

**Comment by katiewalker** [comment]
@JakeLane If this is all good, we can publish tonight.

File: lightning_quacks.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="canonical" href="https://csclub.org.au/about" />
```
Comment (JakeLane) [inline lightning_quacks.html]: `<link rel="canonical" href="https://csclub.org.au/lightning_quacks" />`

File: lightning_quacks.html [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        </li>
+        <li class="search">
+            <div class="search-wrapper card">
+                <input class="search" placeholder="Search">
+                <i class="material-icons">search</i>
+                <div class="search-results"></div>
+            </div>
+        </li>
+        <li><a href="/"><i class="material-icons">home</i>Home</a></li>
+        <li class="active"><a href="/about"><i class="material-icons">info</i>About</a></li>
```
Comment (JakeLane) [inline lightning_quacks.html]: `<li><a href="/about"><i class="material-icons">info</i>About</a></li>`

**Review by JakeLane — CHANGES_REQUESTED** [review]
Just a couple of copy paste artefacts, I'll fix em

**Review by JakeLane — APPROVED** [review]

**Pull request merged**